copy_test.go

```diff
@@ -94,6 +94,85 @@ func TestCopyInMultipleValues(t *testing.T) {
 	}
 }
 
+/* Create this function in your DB to test the COPY statement with triggers that use RAISE
+statement to report messages
+
+CREATE OR REPLACE FUNCTION temptest()
+  RETURNS trigger AS
+$BODY$ begin
+  raise notice 'hello world';  
+  return new;
+end; $BODY$
+LANGUAGE plpgsql
+*/
+func TestCopyInRaiseStmtTrigger(t *testing.T) {
+	db := openTestConn(t)
+	defer db.Close()
+
+	txn, err := db.Begin()
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer txn.Rollback()
+
+	_, err = txn.Exec("CREATE TEMP TABLE temp (a int, b varchar)")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	//Check if test function exists 
+	var num int
+	err = txn.QueryRow(`SELECT COUNT(*) FROM pg_proc where proname = 'temptest'`).Scan(&num)
```

Comment:
You should instead create the function here.  You can create functions in the pg_temp schema, and they will act like temporary tables.